examples/java/spring-boot/src/test/java/uk/gov/api/springboot/models/metadata/ApiMetadataTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.api.springboot.models.metadata;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.IOException;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.json.JsonTest;
+import org.springframework.boot.test.json.JacksonTester;
+import org.springframework.boot.test.json.JsonContent;
+
+@JsonTest
+public class ApiMetadataTest {
+
+  @Autowired private JacksonTester<ApiMetadata> jacksonTester;
+
+  private JsonContent<ApiMetadata> jsonContent;
+
+  @Nested
+  class Serialization {
+
+    @Test
+    void apiVersionIsSerialized() throws IOException {
+      ApiMetadata apiMetadata = new ApiMetadata();
+      apiMetadata.setApiVersion("v1-v2");
+      jsonContent = jacksonTester.write(apiMetadata);
+      assertThat(jsonContent).extractingJsonPathStringValue("api-version").isEqualTo("v1-v2");
+    }
+
+    @Test
+    void dataIsSerialized() throws IOException {
+      Data data = new Data();
+      ApiMetadata apiMetadata = new ApiMetadata();
+      apiMetadata.setData(data);
+      jsonContent = jacksonTester.write(apiMetadata);
+      assertThat(jsonContent).hasJsonPathMapValue("data");
+    }
```

Comment:
I'd recommend using line breaks to make this a given-when-then, and make this a little clearer in terms of what each set of steps are doing:

```suggestion
    void apiVersionIsSerialized() throws IOException {
      ApiMetadata apiMetadata = new ApiMetadata();
      apiMetadata.setApiVersion("v1-v2");
      
      jsonContent = jacksonTester.write(apiMetadata);
      
      assertThat(jsonContent).extractingJsonPathStringValue("api-version").isEqualTo("v1-v2");
    }

    @Test
    void dataIsSerialized() throws IOException {
      Data data = new Data();
      ApiMetadata apiMetadata = new ApiMetadata();
      apiMetadata.setData(data);
      
      jsonContent = jacksonTester.write(apiMetadata);
      
      assertThat(jsonContent).hasJsonPathMapValue("data");
    }
```